web/templates/typeahead_list_item.hbs

```diff
@@ -24,7 +24,7 @@
     <span role="button" class="conversation-arrow zulip-icon zulip-icon-chevron-right stream-to-topic-arrow"></span>
 </div>
 <div class="typeahead-text-container">
-    <strong class="typeahead-strong-section">
+    <strong class="typeahead-strong-section {{#if is_empty_string_topic}} empty-topic-display{{/if}}">
         {{~ topic ~}}
     </strong>
 </div>
```

Comment:
We no longer need `{{#if is_empty_string_topic}} empty-topic-display{{/if}}` on line 34 below, right ? -- The `div` in the else block.